lib/_debugger.js

```diff
@@ -282,7 +282,7 @@ Client.prototype.reqLookup = function(refs, cb) {
   this.req(req, function(err, res) {
     if (err) return cb(err);
     for (var ref in res) {
-      if (util.isObject(res[ref])) {
+      if (res[ref] && typeof res[ref] === 'object') {
```

Comment:
'is object?' checks are better written as `x !== null && typeof x === 'object'`.

`x && ...` generates a TO_BOOLEAN_IC whereas `x !== null && ...` compiles down to a simple check against the null singleton.